addons/hw_drivers/websocket_client.py

```diff
@@ -73,15 +75,16 @@ def on_close(ws, close_status_code, close_msg):
     _logger.debug("websocket closed with status: %s", close_status_code)
 
 
+@helpers.require_db
```

Comment:
very sus structure there, can we really use that ? If I understood correctly the class is not defined if there is no database set to the IoT ?